### ISSUE TITLE
DRASTIC MEMORY OVERFLOW and make_triangles (Valve hammer)

### ISSUE THREAD

**Issue by matiasg90** [Facepunch/garrysmod-issues#4352]
since the last update, these errors have appeared inside valve hammer
![gmod](https://gyazo.com/8b6017ee66119f4a568b41f90342d430.png)

![gmod](https://gyazo.com/165c2b50eadbf66622210bab93582bd9.png)

I'm just compiling BSP

**Comment by robotboy655** [comment]
What beta branch are you on? What update specifically? Which tool are you using, which stage of map compilation? VBSP? VVIS? VRAD? Does the same issue happen in HL2/TF2/CSS/CSGO hammer editors?

**Comment by matiasg90** [comment]
I am using 64bits + chromium
I've been using Garrysmod's valve hammer

![gmod](https://gyazo.com/f6b0065b933b4708ac92b99f4e0fcc08.png)
in the basic compilation of VBSP  (only solids to check for errors)

**Comment by MARK2580** [comment]
> I am using 64bits + chromium
> I've been using Garrysmod's valve hammer
> 
> ![gmod](https://camo.githubusercontent.com/78578fe9d2215a193a3ceedddf5ce4ff45306ffe/68747470733a2f2f6779617a6f2e636f6d2f66366230303635623933336234373038616339326239396634653066636330382e706e67)
> in the basic compilation of VBSP (only solids to check for errors)
> 
> **I just tried the stable version of garrysmod, and it works fine, the problem occurs with the 64bits + chromium version**
> 
> when starting hammer with the 64bit version
> ![gmod](https://camo.githubusercontent.com/fa49d247568af411aeab43605bb48c6863876f5e/68747470733a2f2f6779617a6f2e636f6d2f38653736316335663165656465343931653736626463343564336632386665332e706e67)
Yes, this spam in Hammer messages infuriates, it finds some errors from scratch. This is not in the DEV version.

**Comment by robotboy655** [comment]
OK how about you answer the rest of my questions? Does your .vmf work in Hammer from other games? Or even on the default beta branch? What is this "last update"? Have you considered the problem is with your .vmf and not vbsp?

**Comment by thegrb93** [comment]
@matiasg90 send him a .vmf where the problem happens. You can't expect him to find the problem himself when its probably a specific case in your map causing it.
